platform-core-module/src/test/java/com/softicar/platform/core/module/email/buffer/BufferedEmailSenderTest.java

```diff
@@ -0,0 +1,54 @@
+package com.softicar.platform.core.module.email.buffer;
+
+import com.softicar.platform.core.module.email.EmailSystemProperties;
+import com.softicar.platform.core.module.server.AGServer;
+import com.softicar.platform.core.module.test.AbstractCoreTest;
+import com.softicar.platform.db.runtime.utils.DbAssertUtils;
+import org.junit.Test;
+
+public class BufferedEmailSenderTest extends AbstractCoreTest {
+
+	private final AGServer testServer;
+
+	public BufferedEmailSenderTest() {
+
+		this.testServer = insertServer("Server", "1.1.1.1", 25, "domain", "username", "password");
+		System.setProperty(EmailSystemProperties.SENDING_ENABLED.getPropertyName().toString(), "false");
```

Comment:
- modifying these global properties is quite ugly and dangerous (can lead to hard to find bugs in the tests)
- but I also think there is hardly a way around it, without an abstraction-layer

Reply:
suggestion: add a TODO with reference to a story that will fix this through an abstraction layer

Reply:
As I said, this doesnt work.